src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/AssigningSymbolAndItsMemberInSameStatementTests.cs

```diff
@@ -297,18 +297,18 @@ public async Task CSharpReassignLocalValueTypeVariableAndReferToItsProperty()
             await VerifyCS.VerifyAnalyzerAsync(@"
 public struct S
 {
-    public S Property { get; set; }
+    public S {|CS0523:Property|} { get; set; }
 }
 
 public class Test
 {
     public void Method()
     {
         S a, b;
-        a.Property = c = a = b;
+        {|CS0165:a|}.Property = {|CS0103:c|} = a = {|CS0165:b|};
```

Comment:
😕 Why do we have `c` here?

Reply:
No idea :C 
Let me know if you expect some changes here.

Reply:
@mavasani ?

Reply:
I presume it is a typo. @Evangelink can you please remove 'c'?